pkg/blockstorage/azure/azure_instance_metadata.go

```diff
@@ -18,6 +18,8 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"net/http"
+
+	"github.com/pkg/errors"
```

Comment:
Please use the kanister errors package

Reply:
@tdmanv I don't think we have Kanister errors package unless I am missing something. We do have Kanister logging package but not errors

Reply:
I am merging this now, will raise follow up PR based on the discussion